openpype/hosts/blender/plugins/load/load_model.py

```diff
@@ -41,68 +41,77 @@ def _remove(self, asset_group):
                 objects.extend(obj.children)
                 bpy.data.objects.remove(obj)
 
-    def _process(self, libpath, asset_group, group_name):
+    def _process(self, libpath):
+        """Load the blend library file"""
         with bpy.data.libraries.load(
-            libpath, link=True, relative=False
+                libpath, link=True, relative=False
         ) as (data_from, data_to):
-            data_to.objects = data_from.objects
-
-        parent = bpy.context.scene.collection
+            data_to.collections = data_from.collections
 
-        empties = [obj for obj in data_to.objects if obj.type == 'EMPTY']
+        scene_collection = bpy.context.scene.collection
 
+        # Find the loaded collection and call it container
```

Comment:
I think it'd be better to rename `container` to `container_collection` to explicit the name